manifests/packages.pp

```diff
@@ -1,12 +1,15 @@
 class postfix::packages {
   include ::postfix::params
 
-  package { 'postfix':
-    ensure => installed,
+  if ! defined(Package['postfix']) {
```

Comment:
Managing the postfix package resource is totally in the scope of this module, so I can't see the point of checking for its declaration first...